bin/lib/options.js

```diff
@@ -46,7 +54,7 @@ module.exports = [
       'WebID-TLS',
       'WebID-OpenID Connect'
     ],
-    prompt: true,
+    prompt: false,
```

Comment:
Ok. I am confused by the existence of the oidc option if it's not actually implemented.